build/AzurePipelinesTemplates/ProjectReunion-RunHelixTests-Job.yml

```diff
@@ -59,12 +58,43 @@ jobs:
       feedsToUse: config
       nugetConfigPath: nuget.config
       restoreDirectory: packages
+      
+
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
+    displayName: 'NuGet restore MRTCore'
+    inputs:
+      command: 'custom'
+      arguments: 'restore dev\MRTCore\mrt\core\src\packages.config -ConfigFile nuget.config -PackagesDirectory dev\MRTCore\mrt\packages'
```

Comment:
>core\src\packages.config [](start = 42, length = 24)

if your only purpose is to restore TAEF, you picked the wrong one (and made unnecessary update to it)

Reply:
The purpose is to restore Microsoft.Windows.ImplementationLibrary which is necessary to build the test projects.

Reply:
then you are looking for mrt\mrm\unittests\packages.config

---
In reply to: [520095238](https://github.com/microsoft/ProjectReunion/pull/282#discussion_r520095238) [](ancestors = 520095238)